openpype/hosts/max/plugins/publish/validate_renderer_redshift_proxy.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from pymxs import runtime as rt
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.max.api.lib import get_current_renderer
+
+
+class ValidateRendererRedshiftProxy(pyblish.api.InstancePlugin):
+    """
+    Validates Redshift as the current renderer for creating
+    Redshift Proxy
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["redshiftproxy"]
+    hosts = ["max"]
+    label = "Redshift Renderer"
+    actions = [RepairAction]
+
+    def process(self, instance):
+        invalid = self.get_all_renderer(instance)
+        if invalid:
+            raise PublishValidationError("Please install Redshift for 3dsMax"
+                                         " before using this!")
+        invalid = self.get_current_renderer(instance)
+        if invalid:
+            raise PublishValidationError("Current Renderer is not Redshift")
+
+    def get_all_renderer(self, instance):
+        invalid = list()
+        max_renderers_list = str(rt.RendererClass.classes)
+        if "Redshift_Renderer" not in max_renderers_list:
+            invalid.append(max_renderers_list)
+
+        return invalid
+
+    def get_current_renderer(self, instance):
+        invalid = list()
+        renderer_class = get_current_renderer()
+        current_renderer = str(renderer_class).split(":")[0]
```

Comment:
Why do we need to split on `:`. What's the original string value returned by `get_current_renderer`?

Would be great if the [`lib.get_current_renderer` function](https://github.com/ynput/OpenPype/blob/9d3b788fb40137e651273f104682b31f907f2550/openpype/hosts/max/api/lib.py#L130) could maybe have some more info in the docstring too describing what output it generates, maybe with some examples.